packages/react-image/src/components/Image/useImage.ts

```diff
@@ -23,6 +24,26 @@ export const useImage = (
   options: ComposePreparedOptions,
 ): ImageState => {
   const imageRef = React.useRef<HTMLElement>(null);
+  const ariaProps = useImageAria(props);
```

Comment:
I believe this would get more complicated and we will have to have some common hook to resolve these (maybe something similar to the `useAccessibility`, but I am not sure yet.

Reply:
Changes to accessibility will be done in #14506.